app/controllers/forest_liana/smart_actions_controller.rb

```diff
@@ -8,6 +12,25 @@ class SmartActionsController < ForestLiana::ApplicationController
 
     private
 
+    def smart_action_pre_perform_checks
+      get_smart_action_request
+      find_resource
+      check_permission_for_smart_route
+      ensure_record_ids_in_scope
+    end
```

Comment:
You've not added the `signedApprovalRequest` support yet?

Reply:
This is just a simple controller check, the `signedApprovalResquest` support is used on a specific module.